### DISCUSSION TITLE
Does the index need to fit into memory?

### DISCUSSION THREAD

**Discussion by dotgodly** [blevesearch/bleve#1901] [Q&A] [answered]
Or is it streamed from SSD as well?

**Comment by abhinavdangeti** [comment, marked as answer]
Bleve's scorch indexes are disk bound and will use memory maps to load the index in for access. So - no, the entire index does not need to fit into memory.

**Reply by dotgodly** [reply]
Thank you, really enjoying Bleve so far!
